uint16 id #object id
float64 pos_x #coordinates where the object is relative to the map frame
float64 pos_y
float64 pos_z
